ddtrace/tracer.py

```diff
@@ -262,6 +268,7 @@ def __init__(
         self._appsec_processor = None
         self._iast_enabled = config._iast_enabled
         self._endpoint_call_counter_span_processor = EndpointCallCounterProcessor()
+        self._peer_service_enabled = PEER_SERVICE_ENABLED
```

Comment:
nit: was can use the local variable

```suggestion
        self._peer_service_enabled = peer_service_enabled
```